P/polymake/build_tarballs.jl

```diff
@@ -0,0 +1,143 @@
+  ln -s ../config.ninja build/Opt/config.ninja
+  cp ../config/perlx-config-$target.ninja build/perlx/$perl_version/apple-darwin14/config.ninja
+  # for a modified pure perl JSON module and to make miniperl find the correct modules
+  export PERL5LIB=$prefix/lib/perl5/$perl_version:$prefix/lib/perl5/$perl_version/darwin-2level:$WORKSPACE/srcdir/patches
+  atomic_patch -p1 ../patches/polymake-cross.patch
+  atomic_patch -p1 ../patches/polymake-cross-build.patch
+else
+  ./configure CFLAGS="-Wno-error" CC="$CC" CXX="$CXX" \
+              PERL=${prefix}/deps/Perl_jll/bin/perl LDFLAGS="$LDFLAGS" \
+              --prefix=${prefix} \
+              --with-flint=${prefix}/deps/FLINT_jll \
+              --with-gmp=${prefix}/deps/GMP_jll \
+              --with-mpfr=${prefix}/deps/MPFR_jll \
+              --with-ppl=${prefix}/deps/PPL_jll \
+              --with-bliss=${prefix}/deps/bliss_jll \
+              --with-boost=${prefix}/deps/boost_jll \
+              --with-cdd=${prefix}/deps/cddlib_jll \
+              --with-lrs=${prefix}/deps/lrslib_jll \
+              --with-libnormaliz=${prefix}/deps/normaliz_jll \
+              --without-singular \
+              --without-native
+fi
+
+ninja -v -C build/Opt -j${nproc}
+
+ninja -v -C build/Opt install
+
+# undo patch needed for building
+if [[ $target == *darwin* ]]; then
+  atomic_patch -R -p1 ../patches/polymake-cross-build.patch
+fi
+install -m 444 -D support/*.pl $prefix/share/polymake/support/
+
+# replace miniperl
+sed -i -e "s/miniperl-for-build/perl/" ${libdir}/polymake/config.ninja ${bindir}/polymake*
+# replace binary path with env
+sed -i -e "s#$bindir/perl#/usr/bin/env perl#g" ${libdir}/polymake/config.ninja ${bindir}/polymake*
+# remove target and sysroot
+sed -i -e "s#--sysroot[ =]\S\+##g" ${libdir}/polymake/config.ninja
+sed -i -e "s#-target[ =]\S\+##g" ${libdir}/polymake/config.ninja
+
+# copy of build config that has prefix as a variable
+sed -e "s#${prefix}#\${prefix}#g" ${libdir}/polymake/config.ninja > ${libdir}/polymake/config-reloc.ninja
+
+# cleanup symlink tree
+rm -rf ${prefix}/deps
+
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+ MacOS(:x86_64, compiler_abi=CompilerABI(cxxstring_abi=:cxx11)),
+ Linux(:x86_64, libc=:glibc, compiler_abi=CompilerABI(cxxstring_abi=:cxx11)),
+]
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libpolymake", :libpolymake; dont_dlopen=true)
+    LibraryProduct("libpolymake-apps-rt", :libpolymake_apps_rt; dont_dlopen=true)
+    ExecutableProduct("polymake", :polymake)
+    ExecutableProduct("polymake-config", Symbol("polymake_config"))
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("CompilerSupportLibraries_jll")
+    Dependency(PackageSpec(name="FLINT_jll", uuid="e134572f-a0d5-539d-bddf-3cad8db41a82",version=v"2.6.0"))
```

Comment:
Hopefully this can be safely (?) switch to 2.6.2 once https://github.com/JuliaPackaging/Yggdrasil/pull/1365 is merged?